lib/src/values/logic_value.dart

```diff
@@ -928,7 +928,13 @@ abstract class LogicValue {
     ].swizzle();
   }
 
-  ///compare two LogicValues values for some bits, but not all bits.
+  ///The goal here is to compare two LogicValues that are the same width 
```

Comment:
Check out some guidelines on good doc comment writing here:
https://dart.dev/guides/language/effective-dart/documentation#doc-comments

The top line should describe what the function _does_.  In this case, it should be something like:
```dart
Checks if `this` is equal to [other], except ignoring bits of either which are not valid.

Returns `true` if each bit of `this` which [isValid] is equal to each bit of [other] which [isValid].
```

The first line is clear, short, and describes what it's doing.  Further lines below it explain more formally what it is doing without describing the actual implementation.

It would be good then to include some sort of example showing how it works.